src/main/java/com/segment/analytics/android/integrations/adobeanalytics/AdobeIntegration.java

```diff
@@ -88,7 +89,8 @@ public String key() {
       config.trackingServer = settings.getString("heartbeatTrackingServer");
       config.channel = settings.getString("heartbeatChannel");
       // default app version to 0.0 if not otherwise present b/c Adobe requires this value
-      config.appVersion = (!isNullOrEmpty(context.getPackageName())) ? context.getPackageName() : "0.0";
+      config.appVersion =
```

Comment:
I think when ternary lines get too long, this is more readable as a a simple if else.

Reply:
Totally - already made that change in another branch. This branch sprouted off of master before I merged the branch with the if/else statement into master. Thanks!